azure-quantum/azure/quantum/qiskit/backends/ionq.py

```diff
@@ -147,7 +147,7 @@ def __init__(self, name: str, provider: "AzureQuantumProvider", **kwargs):
                 "description": "IonQ Aria QPU on Azure Quantum",
                 "basis_gates": self._basis_gates(),
                 "memory": False,
-                "n_qubits": 23,
+                "n_qubits": 25,
```

Comment:
Looks like there might be a unit test failing tied to asserting this number:
test_qiskit_get_ionq_qpu_target

Reply:
got it thanks!